### PR TITLE
Updated Description for Deployment type EPIC

### PR DESCRIPTION
**Pull request by bhagyagkwd**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/d/pi_instance.html.markdown [hunk 1]
```diff
@@ -53,7 +53,7 @@ In addition to all argument reference list, you can access the following attribu
   - `network_id` - (String) The network ID of the instance.
   - `network_name` - (String) The network name of the instance.
   - `type` - (String) The type of the network.
-- `deployment_type` - (String) Custom deployment type information (For Internal Use Only).
+- `deployment_type` - (String) Custom deployment type; Allowable value: `EPIC`.
```
Comment (rmoralesjr) [inline website/docs/d/pi_instance.html.markdown:56]: Can we change this to 
_The custom deployment type._

**Review by rmoralesjr — CHANGES_REQUESTED** [review]
Just one change for the data source docs. Thanks

File: website/docs/d/pi_instance.html.markdown (hunk 1 above)
Comment (bhagyagkwd) [reply website/docs/d/pi_instance.html.markdown]: done

**Review by rmoralesjr — APPROVED** [review]
Looks good to me. Thanks

**Pull request merged**